packages/api/migrations/20201105010055-modify_user_removing_password.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return Promise.all([
```

Comment:
Nit: I don't think you need the `return Promise.all()` here or line 11, since you're just don't the one operation.